bootloader/wscript

```diff
@@ -629,8 +630,12 @@ def configure(ctx):
             ctx.env.append_value('LINKFLAGS', '-municode')
 
     if ctx.env.DEST_OS == 'darwin':
-        ctx.env.append_value('CFLAGS', '-mmacosx-version-min=10.7')
-        ctx.env.append_value('LINKFLAGS', '-mmacosx-version-min=10.7')
+        if not any(x for x in ctx.env.CPPFLAGS + ctx.env.CFLAGS
+                   if '-mmacosx-version-min=' in x):
```

Comment:
```suggestion
                   if x.startswith('-mmacosx-version-min=')):
```

Reply:
It is not necessary that the value of any of `CPPFLAGS` or `CFLAGS` will only contain `-mmacosx-version-min=` and/or will start with it.

Reply:
Ah, these are lists, my bad.